partials/vue.js

```diff
@@ -177,7 +174,8 @@ const silvermineRulesVue2Only = {};
 
 const silvermineRulesVue3Only = {
    'vue/no-deprecated-v-is': 'error',
-   // 'vue/v-on-event-hyphenation': 'TODO',
+   'vue/multi-word-component-names': 'off',
+   'vue/v-on-event-hyphenation': 'error',
```

Comment:
As you mentioned in the commit message, I think some `kebab-case` event names have entered into our code base. However, I think it would be better to standardize on `camelCase` because:

1. We use `camelCase` for prop names and `TitleCase` for Vue component names even though Vue does kebab-case transformation for those two things, so using `camelCase` here too would be consistent.
2. We emit the events in `camelCase` so, keeping the case consistent in the `@` listeners means you don't have to mentally convert camel to kebab, and have to remember to do so every time.

That said, if you see a strong reason to keep this as 'error', let me know.

Reply:
I understand and agree. A small handful of adjustments to fix the kebab case but then the consistent convention will be worth it. Sorry for not noticing your review sooner. @yokuze